spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -301,7 +302,9 @@ public static Term toIcebergTerm(Expression expr) {
       Transform transform = (Transform) expr;
       Preconditions.checkArgument(transform.references().length == 1,
           "Cannot convert transform with more than one column reference: %s", transform);
-      String colName = DOT.join(transform.references()[0].fieldNames());
+      List<String> fieldNames = Arrays.stream(transform.references()[0].fieldNames())
+              .map(Spark3Util::toIcebergField).collect(Collectors.toList());
```

Comment:
Nit: This is over indented. The dot in `.map` should line up with the first double quote in the new line 304 (so 4 spaces indented past List).

Reply:
done